### PR TITLE
Display Google Map when there is only one person in the session 

### PR DESCRIPTION
**Pull request by Luxi-Zhao**
If no one joins the session, session host will still see a go button. In this case, the go button launches google map to show a driving path for a driver or a bus path for a passenger, rather than launching the algorithm.

### CODE REVIEW

File: app/src/main/java/com/android/ubclaunchpad/driver/session/SessionInfoActivity.java [hunk 1]
```diff
@@ -124,12 +124,67 @@ public void onCancelled(DatabaseError databaseError) {
...
             @Override
             public void onClick(View v) {
-                // Starting the goddamn algorithm
-                setDriverPassengers();
+                // there is only one person in the session
+                if (itemsArray.size() == 1) {
+                    session.addListenerForSingleValueEvent(new ValueEventListener() {
+                        @Override
+                        public void onDataChange(DataSnapshot dataSnapshot) {
+                            SessionModel s = dataSnapshot.getValue(SessionModel.class);
```
Comment (frostyshadows) [inline app/src/main/java/com/android/ubclaunchpad/driver/session/SessionInfoActivity.java]: can you give the SessionModel a more descriptive name?

File: app/src/main/java/com/android/ubclaunchpad/driver/session/SessionInfoActivity.java [hunk 2]
```diff
@@ -124,12 +124,67 @@ public void onCancelled(DatabaseError databaseError) {
...
+
+                } else {
+                    setDriverPassengers();
+                }
             }
         });
     }
 
+    private void openGoogleMap(boolean isDriver, String curLatLng, String destLatLng) {
+        String origin = "&origin=" + curLatLng;
```
Comment (frostyshadows) [inline app/src/main/java/com/android/ubclaunchpad/driver/session/SessionInfoActivity.java]: most of this method should go into the GoogleDirectionsURLEncoder class

**Review by frostyshadows — CHANGES_REQUESTED** [review]
Yay it works! Requesting a few small changes.

File: app/src/main/java/com/android/ubclaunchpad/driver/session/SessionInfoActivity.java (hunk 2 above)
Comment (Luxi-Zhao) [reply app/src/main/java/com/android/ubclaunchpad/driver/session/SessionInfoActivity.java]: should i overload the encodeURL method to take in just origin, destination and travelmode?

Comment (frostyshadows) [reply app/src/main/java/com/android/ubclaunchpad/driver/session/SessionInfoActivity.java]: you can change the current encodeURL() method's name if you want, to make it more clear, then add a second method with a different name

**Review by frostyshadows — APPROVED** [review]

**Pull request merged**